src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/SystemCommandLine/Utils.tt

```diff
@@ -522,6 +510,32 @@ public class OptionData
             }
         }
 
+        return options;
+    }
+
+    void GenerateCustomBinderClass(string className, string binderClassName, IReadOnlyCollection<OptionData> options)
+    {
+        if (className == null)
+        {
+            throw new ArgumentNullException(nameof(className));
+        }
+        if (binderClassName == null)
+        {
+            throw new ArgumentNullException(nameof(binderClassName));
+        }
+        if (options == null)
+        {
+            throw new ArgumentNullException(nameof(options));
+        }
+        if (className.Trim() == string.Empty)
+        {
+            throw new ArgumentException("Trimmed value cannot be empty", nameof(binderClassName));
+        }
+        if (binderClassName.Trim() == string.Empty)
+        {
+            throw new ArgumentException("Trimmed value cannot be empty", nameof(className));
```

Comment:
Same as above, message requires too much knowledge that we're testing a trimmed string. Consider rephrasing to say it can't be whitespace or empty.

Reply:
Ok. Changed to string.IsNullOrWhiteSpace()